src/components/Sponsors/Sponsors.css

```diff
@@ -72,6 +78,28 @@
 	line-height: max(2rem, 2.5vmin);
 }
 
+.button-container {
+	margin: 3vmin auto;
+}
+
+.button-container .inner-button {
```

Comment:
I would recommend not changing the class names and instead just use `.button button` as the selector here